.github/workflows/generate-dsl-info.yaml

```diff
@@ -0,0 +1,43 @@
+name: Generate DSL Info
+
+permissions:
+  contents: write
+
+on:
+  push:
+    branches:
+      - main
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+
+jobs:
+  generate-dsl-info:
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/main'
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: lts/*
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Generate DSL info
+        run: npm run generate-dsl-info
+
+      - name: Commit changes
```

Comment:
https://github.com/stefanzweifel/git-auto-commit-action

Using this action might be more convenient?

Reply:
cool